test-cases/test-cases.md

```diff
@@ -0,0 +1,36 @@
+# Test cases
+
+Some of the test cases below are implemented in
+[cukinia](https://github.com/3mdeb/meta-trenchboot/blob/8b4812d8577c57b2ee9d1b6bf6898b84bb863fe5/recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf).
+Currently, those are tests 5 through 18.
+
+1. UEFI boot works - TrenchBoot boots into OS
+2. Legacy boot works - TrenchBoot boots into OS
+3. Normal boot (`boot` menuentry) works - TrenchBoot boots into OS
+4. SKL boot (`skl-boot` menuentry) works - TrenchBoot boots into OS
```

Comment:
This probably also needs change in grub.cfg:

```suggestion
4. Slaunch boot (`slaunch-boot` menuentry) works - TrenchBoot boots into OS
```

SKL is AMD only, but it should work botn on AMD and Intel.

Reply:
fixed in 52c44ea2724ca1dccd8538032cc9232c0fd569d7. Also changed in grub.cfg